packages/note/src/react/__stories__/index.story.js

```diff
@@ -210,23 +210,21 @@ storiesOf('Note/actions', module)
         <>
           <NoteWithDefaults
             actionBar={[
-              <Below
+              <BelowLeft
                 when={isOpen}
                 show={
-                  <div style={{ position: 'relative' }}>
```

Comment:
@danethurber I believe your Position fix mean we no longer have to wrap ActionMenu